linera-core/benches/client_benchmarks.rs

```diff
@@ -54,39 +54,30 @@ pub async fn run_claim_bench<B>(
     let amt = Amount::ONE;
 
     let account = Account::new(chain2.chain_id(), owner1);
-    let cert = chain1
+    chain1
         .transfer_to_account(AccountOwner::CHAIN, amt, account)
         .await
         .unwrap()
         .unwrap();
 
-    chain2
-        .receive_certificate_and_update_validators(cert)
-        .await
-        .unwrap();
+    chain2.synchronize_from_validators().await.unwrap();
```

Comment:
Unrelated to this PR, but is there a reason why this function does not return a `anyhow::Result<()>` instead of all thses unwrappings?